### PR TITLE
feat: add ci

### PR DESCRIPTION
**Pull request by martabal**
This PR adds a little bit of CI (eslint and prettier).

Other changes:
- added a README
- added a LICENSE
- changed the Dockerfile to use `npm ci` instead of `npm i`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,15 +10,22 @@
...
+    "format:fix": "prettier --write .",
+    "lint": "eslint . --max-warnings 0",
+    "lint:fix": "npm run lint -- --fix"
   },
   "dependencies": {
     "@discordx/importer": "^1.2.3",
     "discord.js": "^14.13.0",
-    "discordx": "^11.7.6"
+    "discordx": "^11.7.6",
+    "eslint": "^8.50.0"
```
Comment (jrasm91) [inline package.json:23]: Should this be a dev-dependency?

Comment (martabal) [reply package.json:23]: forgot the `-D` 😣

Comment (danieldietzler) [reply package.json:23]: @martabal If you fix it, could you also remove the .DS_STORE file I accidentally pushed?

Comment (martabal) [reply package.json:23]: > @martabal If you fix it, could you also remove the .DS_STORE file I accidentally pushed?

Where is it ? I can't find it

Comment (danieldietzler) [reply package.json:23]: In src: https://github.com/immich-app/discord-bot/blob/main/src/.DS_Store

Comment (martabal) [reply package.json:23]: Thanks, I didn't know VS Code hides those files :smile:

**Pull request merged**